### PR TITLE
SE-3016: Add PDF generation scripts and examples

### PR DESCRIPTION
**Pull request by dgbarclay**

### CODE REVIEW

**Review by steco — APPROVED** [review]

File: examples/lusid/pdf-generation/2-generate-pdf.sql [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Lusid.Portfolio.Holding h
+where
+h.LusidInstrumentId = t.LusidInstrumentId
+and
+h.PortfolioScope = @@portfolio_scope 
+and 
+h.PortfolioCode = @@portfolio_code;
+
+
+-- Get holdings and format
```
Comment (steco) [inline examples/lusid/pdf-generation/2-generate-pdf.sql:44]: Comment

**Comment by slemasne** [comment]
Tests passed:
https://github.com/finbourne/luminesce-examples/actions/runs/4366848399

**Pull request merged**